content/en/docs/reference/terms/index.md

```diff
@@ -144,6 +148,10 @@ see [_Device-bound passkey_.](#device-bound-passkey)
 
 A FIDO2 [Discoverable Credential](#discoverable-credential) that can reliably be used for bootstrapping sign-in, without requiring other login challenges such as passwords and OTPs. "Reliable" here means that the passkey should be available to, and usable by, the user whenever they need to sign in. This availability can be achieved through different means: for example, passkey providers could sync passkeys in real-time across a user's devices, restore passkeys from a backup whenever a user sets up a new device, offer passkeys across different contexts (a passkey established from an app can be used in the browser when visiting the app’s website), or allow users to [exercise passkeys across devices](#cross-device-authentication-cda) (by, say, using the passkey from a nearby phone when signing in from a laptop).
 
+## Third-Party Passkey Provider
+
+A [Passkey Provider](#passkey-provider) that is not built-in to the device's operating system and plugs in to the OS via platform APIs.
```

Comment:
Maybe something like this?

```suggestion
A [Passkey Provider](#passkey-provider) that plugs in to the OS via platform APIs to store and manage a user's passkeys via the platform authentication.
```

Reply:
Addressed in [e6ba66a](https://github.com/passkeydeveloper/passkeys.dev/pull/215/commits/e6ba66a2af6b7601bb11c893223037b1094499ef)